### PR TITLE
[docs]:open studio from local deployment

### PR DESCRIPTION
**Pull request by isahers1**

### CODE REVIEW

File: docs/docs/cloud/how-tos/test_local_deployment.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Test Local Deployment
```
Comment (hwchase17) [inline docs/docs/cloud/how-tos/test_local_deployment.md:1]: title should focus on studio more

File: docs/docs/cloud/how-tos/test_local_deployment.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+!!! warning "Browser Compatibility"
+    Viewing the studio page of a local LangGraph deployment does not work in Safari. Use Chrome instead.
+
+## Setup
+
+Make sure you have [setup your app](https://langchain-ai.github.io/langgraph/cloud/deployment/setup/) correctly.
+
+After you have your app setup, [start the API server](https://langchain-ai.github.io/langgraph/cloud/deployment/test_locally/#start-the-api-server) and ensure that it is working properly.
```
Comment (hwchase17) [inline docs/docs/cloud/how-tos/test_local_deployment.md:10]: we should tell them to start it in watch mode to pick up code changes

Comment (hwchase17) [reply docs/docs/cloud/how-tos/test_local_deployment.md:10]: you should put any commands they need to run in this page - they shouldnt have to keep jumping back and forth

File: docs/docs/cloud/how-tos/test_local_deployment.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Test Local Deployment
+
+!!! warning "Browser Compatibility"
+    Viewing the studio page of a local LangGraph deployment does not work in Safari. Use Chrome instead.
+
+## Setup
+
+Make sure you have [setup your app](https://langchain-ai.github.io/langgraph/cloud/deployment/setup/) correctly.
```
Comment (hwchase17) [inline docs/docs/cloud/how-tos/test_local_deployment.md:8]: id put some of the basics here so they dont have to jump back and forth

**Comment by isahers1** [comment]
thanks for the feedback, latest commit should have addressed comments

File: docs/docs/cloud/how-tos/index.md [hunk 4]
```diff
@@ -52,6 +52,7 @@ When creating complex graphs, leaving every decision up to the LLM can be danger
 LangGraph Studio is a built-in UI for visualizing, testing, and debugging your agents.
 
 - [How to enter LangGraph Studio](./test_deployment.md)
+- [How to enter LangGraph Studio for local deployment](./test_local_deployment.md)
```
Comment (vbarda) [inline docs/docs/cloud/how-tos/index.md:55]: would be nice for these titles to match the ones in mkdocs

**Pull request merged**